### PR TITLE
fix#219: code cleanup: move printver.tcl into info-command.tcl

### PR DESCRIPTION
**Pull request by ItsAJ1005**
It seems that https://github.com/valkey-io/valkey/blob/unstable/tests/unit/printver.tcl should be absorbed into https://github.com/valkey-io/valkey/blob/unstable/tests/unit/info-command.tcl

It was created 13 years ago by antirez. Based on the comment

Print version info before running the test
it was created as a development test tool just for the two info-fields redis_version and redis_git_sha1 (which is now server_git_sha1). All it does is check that the fields are present and print their values.

redis_version is already checked in info-command.tcl. redis_git_sha1 and some other fields should be added to info-command.tcl.

### CODE REVIEW

**Comment by ItsAJ1005** [comment]
@daniel-house Is my method correct? Please review my and code and provide suggestions...😃

File: tests/unit/printver.tcl [hunk 1]
```diff
@@ -1,6 +0,0 @@
-start_server {} {
```
Comment (madolson) [inline tests/unit/printver.tcl:1]: I think we can just delete this file? I don't think there is any real value in it anymore.

Comment (ItsAJ1005) [reply tests/unit/printver.tcl:1]: But in the description they mentioned "Print version info before running the test
it was created as a development test tool just for the two info-fields redis_version and redis_git_sha1 (which is now server_git_sha1)."

Comment (hwware) [reply tests/unit/printver.tcl:1]: I suggest removing the printver.tcl file directly, it makes no sense now. There is no any function test in this file.

Comment (ItsAJ1005) [reply tests/unit/printver.tcl:1]: Okay will do that 👍

**Comment by hwware** [comment]
Is there any reason close this PR? I am confused...

**Comment by ItsAJ1005** [comment]
> Is there any reason close this PR? I am confused...

There are some signature overlaps. I will reopen it again after resolving then.

**Pull request closed without merging**